Assets/Scripts/Managers/Stats.cs

```diff
@@ -63,9 +79,9 @@ public float SpawnChance(Guild guild)
         public int Defence => Manager.Adventurers.Available + GetStat(Stat.Defence);
 
         // TODO: How do we make a more engaging way to determine threat than just a turn counter?
```

Comment:
I feel like this kind of comment isn't code related. TODOs should generally be about the code.